pinecone/sparse/splade/splade_vector_generation.ipynb

```diff
@@ -0,0 +1,8784 @@
+{
```

Comment:
<p>"In the following example[,] we will use(.) [the] SPLADE Model: <a href="https://huggingface.co/naver/splade-cocondenser-ensembledistil" target="_blank">naver/splade-cocondenser-ensembledistil</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>